pkg/ui/workspaces/cluster-ui/src/databasesPage/databasesPage.tsx

```diff
@@ -536,111 +539,124 @@ export class DatabasesPage extends React.Component<
     return cell;
   };
 
-  private columns: ColumnDescriptor<DatabasesPageDataDatabase>[] = [
-    {
-      title: (
-        <Tooltip placement="bottom" title="The name of the database.">
-          Databases
-        </Tooltip>
-      ),
-      cell: database => (
-        <Link
-          to={EncodeDatabaseUri(database.name)}
-          className={cx("icon__container")}
-        >
-          <StackIcon className={cx("icon--s", "icon--primary")} />
-          {database.name}
-        </Link>
-      ),
-      sort: database => database.name,
-      className: cx("databases-table__col-name"),
-      name: "name",
-    },
-    {
-      title: (
-        <Tooltip
-          placement="bottom"
-          title="The approximate total disk size across all table replicas in the database."
-        >
-          Size
-        </Tooltip>
-      ),
-      cell: database =>
-        this.checkInfoAvailable(database, format.Bytes(database.sizeInBytes)),
-      sort: database => database.sizeInBytes,
-      className: cx("databases-table__col-size"),
-      name: "size",
-    },
-    {
-      title: (
-        <Tooltip
-          placement="bottom"
-          title="The total number of tables in the database."
-        >
-          Tables
-        </Tooltip>
-      ),
-      cell: database => this.checkInfoAvailable(database, database.tableCount),
-      sort: database => database.tableCount,
-      className: cx("databases-table__col-table-count"),
-      name: "tableCount",
-    },
-    {
-      title: (
-        <Tooltip
-          placement="bottom"
-          title="The total number of ranges across all tables in the database."
-        >
-          Range Count
-        </Tooltip>
-      ),
-      cell: database => this.checkInfoAvailable(database, database.rangeCount),
-      sort: database => database.rangeCount,
-      className: cx("databases-table__col-range-count"),
-      name: "rangeCount",
-    },
-    {
-      title: (
-        <Tooltip
-          placement="bottom"
-          title="Regions/Nodes on which the database tables are located."
-        >
-          {this.props.isTenant ? "Regions" : "Regions/Nodes"}
-        </Tooltip>
-      ),
-      cell: database =>
-        this.checkInfoAvailable(
-          database,
-          database.nodesByRegionString || "None",
+  private renderDatabaseNameCell = (
+    database: DatabasesPageDataDatabase,
+  ): React.ReactNode => {
+    const isCockroachCloud = useContext(CockroachCloudContext);
+    const linkURL = isCockroachCloud
+      ? `${location.pathname}/${database.name}`
+      : EncodeDatabaseUri(database.name);
+    return (
+      <Link to={linkURL} className={cx("icon__container")}>
+        <StackIcon className={cx("icon--s", "icon--primary")} />
+        {database.name}
+      </Link>
+    );
+  };
+
```

Comment:
Can this be extracted into its own component?

Reply:
Done, extracted to a `util` file in `databasesPage` directory.